update-dois/src/main/java/org/reactome/release/updateDOIs/UpdateDOIs.java

```diff
@@ -31,10 +34,16 @@ public static void setAdaptors(MySQLAdaptor adaptorTR, MySQLAdaptor adaptorGK) {
 	}
 
 	@SuppressWarnings("unchecked")
-	public static void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pathToReport, boolean testMode) {
+	public static void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pathToReport, String releaseNumber, boolean testMode) throws IOException {
 
+		String doisListFilename = "doisToBeUpdated-v" + releaseNumber + ".txt";
+		File doisListFile = new File(doisListFilename);
 		if (testMode) {
 			logger.info("Test mode is active. Outputting DOIs that can be updated");
+			if (doisListFile.exists()) {
+				doisListFile.delete();
+			}
```

Comment:
You could replace with the Files.deleteIfExists(...) method: https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#deleteIfExists(java.nio.file.Path)